tests/components/zha/test_discover.py

```diff
@@ -281,7 +277,7 @@ def test_discover_probe_single_cluster():
         map_id = (comp, unique_id)
         assert map_id in device_info["entity_map"]
         entity_info = device_info["entity_map"][map_id]
-        assert set([ch.name for ch in channels]) == set(entity_info["channels"])
+        assert {ch.name for ch in channels} == set(entity_info["channels"])
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5306536)